.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+  Publish-Package:
+    if: github.event_name == 'release'
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    needs:
+      - Static-Check
+      - Run-Unit-Tests
+      - Code-Coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
```suggestion
      - uses: actions/checkout@v4
      - uses: actions/setup-python@v4
```

Reply:
setup-python has v5 available

Reply:
Thanks a lot, @pankajastro and @pankajkoti  - I addressed the feedback in 667d34309a96c8d6b73190d53e45f9fe860444ad